### PR TITLE
[DOCS] Creates custom landing page for the Py client book

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR adds a new landing page to the Python client book so it will be more in line with the [main docs landing page](https://www.elastic.co/guide/index.html), the [ES guide landing page](https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html), and the [Kibana guide landing page](https://www.elastic.co/guide/en/kibana/current/index.html).

* [Current Python docs landing page](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/index.html)
* [New Python docs landing page proposed in this PR](https://elasticsearch-py_2187.docs-preview.app.elstc.co/guide/en/elasticsearch/client/python-api/master/index.html)

The content can be easily modified based on feedback.

### CODE REVIEW

**Comment by szabosteve** [comment]
@elasticmachine test this please

File: docs/guide/index-custom-title-page.html [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+  <div class="legalnotice"></div>
+
+  <div class="row my-4">
+    <div class="col-md-6 col-12">
+      <p></p>
+      <p>
+        <h2>Documentation</h2>
+      </p>
+      <p>
+        The official Python client provides a low-level client for communicating with an Elasticsearch cluster.
```
Comment (sethmlarson) [inline docs/guide/index-custom-title-page.html:63]: Could we change the description here? I want to get away from the whole "low-level" versus "high-level" distinction, especially for Python.

Comment (szabosteve) [reply docs/guide/index-custom-title-page.html:63]: @sethmlarson Sure! Do you have any preferred alternative?

Comment (sethmlarson) [reply docs/guide/index-custom-title-page.html:63]: Maybe can copy JS with: "The official Python client provides one-to-one mapping with Elasticsearch REST APIs."

**Pull request merged**